src/utils.c

```diff
@@ -352,3 +353,13 @@ timestamptz_bucket(PG_FUNCTION_ARGS)
 	}
 	PG_RETURN_TIMESTAMPTZ(result);
 }
+
+inline bool
```

Comment:
general question: is this how other postgres options are toggled? on/off vs true/false?